src/openloco/input/mouse_input.cpp

```diff
@@ -1436,8 +1436,17 @@ namespace openloco::input
                 break;
 
             case ui::scrollview::scroll_part::vscrollbar_button_top:
-                call(0x4c8b26, regs);
+            {
+                ui::window* w = WindowManager::find(_pressedWindowType, _pressedWindowNumber);
+                if (w != nullptr)
+                {
+                    w->scroll_areas[scrollAreaIndex].flags |= scroll_flags::VSCROLLBAR_UP_PRESSED;
+                    w->scroll_areas[scrollAreaIndex].v_top = std::max(w->scroll_areas[scrollAreaIndex].v_top - 3, 0);
```

Comment:
We keep using `3`. It should be defined as the amount that the scroll bar moves on a click of the button (in less words...)

Reply:
I can think about some solution in some later branch.